src/install/install.zig

```diff
@@ -7659,8 +7660,13 @@ pub const PackageManager = struct {
                     this.enable.force_save_lockfile = true;
                 }
 
-                this.update.development = cli.development;
-                if (!this.update.development) this.update.optional = cli.optional;
+                if (cli.development) {
```

Comment:
i wonder if `bun add react --peer --optional` is expected to work?

Reply:
what would that do though, last flag used or add to both?

Reply:
We could have it add
```json
"peerDependencies": {
    "react": "<version>"
},
"peerDependenciesMeta": {
    "react": {
        "optional": true
    }
}
```

Reply:
oh i didn't know about this, that's cool

i think it makes sense to add